src/raw/mod.rs

```diff
@@ -181,11 +143,6 @@ impl TableLayout {
     }
 }
 
-/// A reference to an empty bucket into which an can be inserted.
```

Comment:
I decided to delete `InsertSlot` because I think it's a bad API: there's not much stopping you from creating an invalid insertion slot, and it doesn't really gain you anything by making the type different from an ordinary index. We can revisit this type of API for `HashTable` if it seems worthwhile in the future.